File: week1/compare.c
#include <cs50.h>
#include <stdio.h>

void compare(int a, int b);

int main(void)
{
    int x = get_int("Enter First Number: ");
    int y = get_int("Enter Second Number: ");

    compare(x, y);
}

void compare(int a, int b)
{
    if (a > b)
    {
        printf("%i is greater than %i\n", a, b);
    }
    else if (a < b)
    {
        printf("%i is less than %i\n", a, b);
    }
    else
    {
        printf("%i is equals to %i\n", a, b);
    }
}

File: week6/compare.py
# compare two numbers
from cs50 import get_int

n1 = get_int("Number 1: ")
n2 = get_int("Number 2: ")

if n1 > n2:
    print(f"{n1} is greater than {n2}")
elif n1 < n2:
    print(f"{n1} is less than {n2}")
else:
    print(f"{n1} is equal to {n2}")
